lib/fastlane/plugin/sentry/actions/sentry_create_deploy.rb

```diff
@@ -0,0 +1,103 @@
+module Fastlane
+  module Actions
+    class SentryCreateDeployAction < Action
+      def self.run(params)
+        require 'shellwords'
+
+        Helper::SentryHelper.check_sentry_cli!
+        Helper::SentryConfig.parse_api_params(params)
+
+        version = params[:version]
+        version = "#{params[:app_identifier]}@#{params[:version]}" if params[:app_identifier]
+
+        has_time = !params[:time].nil?
+        has_started = !params[:started].nil?
+        has_finished = !params[:finished].nil?
+
+        if !has_time && (has_started && !has_finished || !has_started && has_finished)
+          UI.user_error!("Only one parmeter of 'started' and 'finished' found, please provide both.")
+        end
+
+        command = [
+          "sentry-cli",
+          "releases",
+          "deploys",
+          version,
+          "new"
+        ]
+        command.push('--env').push(params[:env]) unless params[:env].nil?
+        command.push('--name').push(params[:name]) unless params[:name].nil?
+        command.push('--url').push(params[:deploy_url]) unless params[:deploy_url].nil?
+        command.push('--started').push(params[:started]) unless params[:started].nil? || has_time
+        command.push('--finished').push(params[:finished]) unless params[:finished].nil? || has_time
+        command.push('--time').push(params[:time]) unless params[:time].nil?
+
+        Helper::SentryHelper.call_sentry_cli(command)
+        UI.success("Successfully created deploy: #{version}")
+      end
+
+      #####################################################
+      # @!group Documentation
+      #####################################################
+
+      def self.description
+        "Creates a new release deployment for a project on Sentry"
+      end
+
+      def self.details
+        [
+          "This action allows you to accociate deploys to releases for a project on Sentry.",
+          "See https://docs.sentry.io/product/cli/releases/#creating-deploys for more information."
+        ].join(" ")
+      end
+
+      def self.available_options
+        Helper::SentryConfig.common_api_config_items + [
+          FastlaneCore::ConfigItem.new(key: :version,
+                                       description: "Release version to accociate the deloy with on Sentry"),
+          FastlaneCore::ConfigItem.new(key: :env,
+                                       short_option: "-e",
+                                       description: "Set the environment for this release. This argument is required. Values that make sense here would be 'production' or 'staging'",
```

Comment:
`l`: I think it would make sense to add a `.` to finish the sentences in all descriptions here as well for the ReadMe.
```suggestion
                                       description: "Set the environment for this release. This argument is required. Values that make sense here would be 'production' or 'staging'.",
```

Reply:
Actually tried this, but fastlane is reporting this as an error:

```
FastlaneCore::Interface::FastlaneError:
Do not let descriptions end with a '.', since it's used for user inputs as well for key :env
```